eth/tracers/api.go

```diff
@@ -778,7 +778,8 @@ func (api *API) standardTraceBlockToFile(ctx context.Context, block *types.Block
 		// Note: This copies the config, to not screw up the main config
 		chainConfig, canon = overrideConfig(chainConfig, config.Overrides)
 	}
-	evm := vm.NewEVM(vmctx, statedb, chainConfig, vm.Config{})
+
+	evm := vm.NewEVM(vmctx, statedb, api.backend.ChainConfig(), vm.Config{})
```

Comment:
Why to use `api.backend.ChainConfig()` here? Rather than the overridden one

Reply:
thx for the catch! fixed